CONTRIBUTING.md

```diff
@@ -13,6 +13,7 @@ Consult the Table of Contents below, and jump to the desired section.
 - [Where to seek for help?](#where-to-seek-for-help)
   - [Enterprise Edition](#enterprise-edition)
   - [Community Edition](#community-edition)
+- [Our pledge to the community](#our-pledge-to-the-community)
 - [Where to report bugs?](#where-to-report-bugs)
 - [Where to submit feature requests?](#where-to-submit-feature-requests)
 - [Contributing](#contributing)
```

Comment:
Just to add to this PR, I also want to include some suggestions for the contributors (maybe "best practices"?).
For example, smaller PRs are more likely to get reviewed soon. And maybe we could reveal some of our processes when reviewing the RP.

Reply:
> Feel free to contribute fixes or minor features by opening a Pull
> Request.  Small contributions are more likely to be merged quicker
> than changes which require a lot of time to review.  If you are
> planning to develop a larger feature, please talk to us first in the
> [GitHub Discussions](https://github.com/Kong/kong/discussions)
> section!

Does this suffice?